runtime/plugins/comment/comment.lua

```diff
@@ -191,7 +206,7 @@ function string.starts(String,Start)
     return string.sub(String,1,string.len(Start))==Start
 end
 
-function init()
+functioninit()
```

Comment:
What happened here?

Reply:
To be honest, I have no idea how I didn't see this.